### ISSUE TITLE
Dont require github.com token

### ISSUE THREAD

**Issue by sta-szek** [Azure/setup-helm#52]
hello, thanks for this action! 🤝  
it saved some hours of my work 🍻 

however, i have small request.

would it be possible to change the implementation, not to use github.com token for the github api?

i mean this piece of code:

```
async function getLatestHelmVersionFor(type: string): Promise<string> {
    const token = core.getInput('token', { 'required': true });
```

the problem is that we run the action hundreds of thousands of times every day (we have bilions of microservices, each will run it at least 3 times on single PR! and bunch of active developers that are creating PRs every second)

all that, in conjunction, is creating massive attak on github API, which requires github.com token -> as a result, the token is API Rate limited :/ 

can we somehow use `latest` as version, to fetch latest, but do not query github api?

as a result we have blocked pipelines, which is defniately not the way to go.
also, we can consider changing version to some fixed version, but then we need to update the version manually, which is also not the way to go :(

**Comment by qpetraroia** [comment]
Hi @sta-szek, 

We currently have an engineer who is refactoring this action. We will continue to update you with our progress as we move forward.

**Comment by OliverMKing** [comment]
Fixed with #55.
